data/discovery/field/website.yaml

```diff
@@ -2,5 +2,5 @@ cardinality: '1'
 datatype: string
 field: website
```

Comment:
Well thats a sign of a bigger problem, fix it if you can or leave it and create a card